src/components/Message.js

```diff
@@ -0,0 +1,27 @@
+import React, {Component} from 'react'
+import'./css/Message.css'
+
+class Message extends Component{
```

Comment:
could you use function based components

Reply:
Yes i Can use function based components.